### DISCUSSION TITLE
How to change app color scheme?

### DISCUSSION THREAD

**Discussion by mawkler** [KRTirtho/flemozi#44] [Q&A]
Hi! I couldn't find any way of changing the default skin tone from the yellow one. This would be an awesome feature in my opinion!

**Comment by KRTirtho** [comment]
Go to "settings" > "appearance"  and turn off "Sync album color" toggle
Above that you can find "Accent color" where you can change the color scheme

**Reply by mawkler** [reply]
Thank you for your response! However, I'm not sure how to navigate to "appearance". This is what the settings page looks like to me. I'm on v0.2.0.

![image](https://github.com/user-attachments/assets/effa8fa1-73fa-4f94-a101-fd318039c64a)

> where you can change the color scheme

Note that what I want to change is the default skin color of emoji, i.e. change ✋ to ✋🏻 (for all emoji.

**Reply by KRTirtho** [reply]
Lol wrong app 🤣 I was thinking I was replying on my Spotube app's issues
Actually currently flemozi only follows system theme so it's not possible to change the color scheme

**Reply by mawkler** [reply]
I'm not asking about changing the color scheme. I'm not sure why the title says "color scheme" because in [my original issue](https://github.com/KRTirtho/flemozi/issues/43) that was moved here I wrote "skin tone". What I want is to change the default yellow skin tone for emoji.

To illustrate, I want to change ✋ to ✋🏻 (for all emoji) 🙂
